redwood-treehouse/src/hostMain/kotlin/app/cash/redwood/treehouse/TreehouseLauncher.kt

```diff
@@ -80,19 +80,26 @@ public class TreehouseLauncher internal constructor(
     scope.launch(dispatchers.zipline) {
       val ziplineFileFlow = ziplineFlow(spec)
       ziplineFileFlow.collect {
-        val app = spec.create(it.zipline)
-        treehouseApp.onCodeChanged(it.zipline, app)
+        when (it) {
+          is LoadResult.Success -> {
+            val app = spec.create(it.zipline)
+            treehouseApp.onCodeChanged(it.zipline, app)
+          }
+          is LoadResult.Failure -> {
+            // TODO
```

Comment:
I think we can ignore these. Or maybe log them. It’s just gonna be a connectivity error.